composer.json

```
@@ -12,6 +12,9 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
+    "scripts": {
+        "docs": "composer --work-directory docs update && ./docs/vendor/bin/build-docs.sh @additional_args"
```

Sorry, I made a mistake and abbreviated the wrong word 😓 

```suggestion
        "docs": "composer --working-dir docs update && ./docs/vendor/bin/build-docs.sh @additional_args"
```